src/js/glowingbear.js

```diff
@@ -663,9 +663,20 @@ weechat.controller('WeechatCtrl', ['$rootScope', '$scope', '$store', '$timeout',
         $rootScope.hostInvalid = false;
 
         var parts;
+        var regexProto = /^(https?|wss?):\/\/(.+)$/;
         var regexHost = /^([^:\/]*|\[.*\])$/;
         var regexHostPort = /^([^:]*|\[.*\]):(\d+)$/;
-        var regexHostPortPath = /^([^:]*|\[.*\]):(\d*)\/(.+)$/;
+        var regexHostPortPath = /^([^:]*|\[.*\]):(\d+)\/(.+)$/;
```

Comment:
We have to fix a little bug in the host:port/path regex to prevent "wss:/foo.bar" from being turned into host `wss`, port `<empty string>`, path `foo.bar` by requiring at least one digit between `:` and `/`